purview/api/v1.py

```diff
@@ -17,14 +17,25 @@
 @api_v1.route('/about')
 def about():
     """
-    Display version info about purview
-    :return: A JSON object with version info
+    Display general information about the app.
```

Comment:
How about "...about the API" ?

Reply:
Well we're actually exposing the app's version not the API so I think it's good as is.